internal/pkg/model/object.go

```diff
@@ -321,6 +321,10 @@ func (v Blocks) Clone() Blocks {
 
 func (c *Code) Clone() *Code {
 	clone := *c
+	clone.Scripts = make([]string, len(c.Scripts))
```

Comment:
`Code.Script` su typu` []string`, takze na prvy pohlad sa skopiruju, kedze tam nie je pointer.
Ale to nie je pravda. `slice` v go sa vzdy odovzdava ako referencia.
Resp. vzdy existuje nejake `array` na ktore `slice` ukazuje, a ked sa slice (nejaka hlavicka, `start, length`) skopiruje, tak pole sa nekopiruje.

Vid priklad: https://play.golang.org/p/fBs0yixVCwL